cpp/CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ set(KNOWN_WARNINGS
        -Wno-error=unused-variable \
        -Wno-error=uninitialized \
        -Wno-error=maybe-uninitialized \
+       -Wno-unknown-warning-option \
```

Comment:
Will adding it directly like this cause any problems with GCC 13?

Reply:
So `-Wno-unknown-warning-option` doesn't affect GCC 13, it just doesn't work, right?

Reply:
Thanks for your explanation. I'll test in Mac again.